types/v1/test-type/index.d.ts

```diff
@@ -96,3 +109,41 @@ export interface SpecialistCustomDefectsSchema {
    */
   hasAllMandatoryFields?: boolean;
 }
+export interface SpecialistCustomDefectsSchema1 {
```

Comment:
Is there a nicer way of differentiating this from SpecialistCustomDefectsSchema?